Style-Guide/Code-Layout-and-Formatting.md

```diff
@@ -117,28 +138,38 @@ $MyObj.GetData($Param1,
                $Param2,
                $Param3,
                $Param4)
-```
 
+# This is also good
```

Comment:
The point of this section about **hanging indents**  or continuation lines is that it is _ok_ to **not** use a single level 4-space indentation.

Adding an example that is according to the _normal_ standard seems redundant, but it would at _least_ need to be explicitly called out that it's here for contrast:  "According to normal indentation rules, you would write it this way ... "

Reply:
Given that I saw the need to add a 'normal' example, perhaps the intent needs clarification? ;)

I'll review and see what I can do.

Reply:
Perhaps it should be reworded to say _more than_ earlier in the paragraph, and to be clearer about the exception cases. Maybe we could put a simple example of indented code before the second paragraph, and then change that paragraph to read like:

> ##### Use four *spaces* per indentation level
>
> Usually you use the `[Tab]` key to indent, but most editors can be configured to insert spaces instead of actual tab characters when you indent. For most programming languages and editors (including PowerShell ISE) the default is four spaces, and that's what we recommend. As always, individuals, teams and projects may have different standards and you should abide by them...
>
> ```posh
> function Test-Code {
>     foreach ($exponent in 1..10) {
>         [Math]::Pow(2, $exponent)
>     }
> }
> ```
>
> Indenting more than 4-spaces is acceptable for continuation lines (when you're wrapping a line which was too long). In such cases you might indent more than one level, or even indent indent an odd number of spaces to line up with a method call or parameter block on the line before.
>
> ```posh
> function Test-Code {
>     foreach ($base in 1,2,4,8,16) {
>         foreach ($exponent in 1..10) {
>             [System.Math]::Pow($base,
>                                $exponent)
>     }
> }
> ```

Reply:
That does look a lot clearer, thank you!